.circleci/config.yml

```diff
@@ -123,10 +123,19 @@ workflows:
   version: 2
   build-test-deploy:
     jobs:
-      - checkout_code
+      - checkout_code:
+          filters:
+            tags:
+              only: /.*/
       - build_test_and_deploy:
           requires:
             - checkout_code
+          filters:
+            tags:
+              only: /.*/
       - run_lintcheck:
           requires:
             - checkout_code
+          filters:
```

Comment:
`run_lintcheck` should only be run on pull requests.

Reply:
So, ultimately we want the filter to exclude tags *and* `master` but run on all others.